cmd/openfga/openfga.go

```diff
@@ -203,3 +207,28 @@ func buildService(logger logger.Logger) (*service, error) {
 		authenticator: authenticator,
 	}, nil
 }
+
+func buildLogger() (*logger.ZapLogger, error) {
```

Comment:
Why not `logger.Logger`? We should return the interface, not the concrete type here. That way we can change the concrete type that we return in the future if we so choose.

Reply:
[whenever you notice something like that, a wizard did it](https://www.youtube.com/watch?v=sVgVB3qsySQ)